### ISSUE TITLE
Toast positioning

### ISSUE THREAD

**Issue by zensabbah** [rocketlaunchr/electron-alert#7]
Hi, I have problem with Toast.
I saw I can call it from Alert promise and this is wonderful, but I don't know why, any Toast appears in top-start (I think) whatever position I give (even without set it).

Which could be the problem?

**Comment by pjebs** [comment]
Show me how you are calling it?

**Comment by pjebs** [comment]
and a screenshot of the behaviour

**Comment by zensabbah** [comment]
Sorry for the delay!
<img width="1033" alt="Schermata 2020-03-29 alle 13 23 13" src="https://user-images.githubusercontent.com/32987121/77847908-d1753380-71c0-11ea-8a86-20fe6363108a.png">
<img width="340" alt="Schermata 2020-03-29 alle 13 23 54" src="https://user-images.githubusercontent.com/32987121/77847911-d4702400-71c0-11ea-9bf7-bbcf25693ad9.png">
<img width="273" alt="Schermata 2020-03-29 alle 13 24 05" src="https://user-images.githubusercontent.com/32987121/77847912-d4702400-71c0-11ea-96de-d0ca89358a12.png">

**Comment by pjebs** [comment]
Position must be as per sweet alert docs.

**Comment by zensabbah** [comment]
Whatever position I put is the same..I tried now with

`  position: 'bottom-end',
`
and it appears in the same place

**Comment by pjebs** [comment]
```
		let positions = {
			top: "topCenter",
			"top-start": "topLeft",
			"top-end": "topRight",
			center: "center",
			"center-start": "leftCenter",
			"center-end": "rightCenter",
			bottom: "bottomCenter",
			"bottom-start": "bottomLeft",
			"bottom-end": "bottomRight"
		};

		if (swalOptions.position) {
			this.position = positions[swalOptions.position] || "center";
			delete swalOptions.position;
		}

		if (!(isMac && (parent !== undefined && parent !== null))) {
			new Positioner(this.browserWindow).move(this.position);
		}
```

**Comment by pjebs** [comment]
Is `if (!(isMac && (parent !== undefined && parent !== null))) {` satisfied for you?

**Comment by zensabbah** [comment]
I'm sorry but in which function I should put that code? I substitute  `swalOptions ` with my `swalToastCancelOptions`

and added `const isMac = process.platform === 'darwin'

but I did'nt defined `parent`

Is this variable under mainwindow?

`

**Comment by pjebs** [comment]
Is existing code

**Comment by pjebs** [comment]
I'm wondering which execution path it takes

**Comment by zensabbah** [comment]
My code is in main.js in a ipcMain.on.

I solved sending back to renderer a message and using Swal from the other side, but it's really useful your module as is possible to call the alert from the main process.

It's a pity the toast is this strange behaviour but thanks a lot the same for your time and work!!

**Comment by pjebs** [comment]
@zensabbah This issue has been resolved in `v0.1.17`.
It was a major bug in my code.

**Comment by zensabbah** [comment]
Ok, thanks a lot!
